sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -1245,6 +1245,23 @@ public void setSampleAssociations(EventParticipantReferenceDto sourceEventPartic
 		}
 	}
 
+	@Override
+	public void setSampleAssociationsUnrelatedDisease(EventParticipantReferenceDto sourceEventParticipant, CaseReferenceDto cazeRef) {
+		if (sourceEventParticipant != null) {
```

Comment:
I'd remove the null check - users of the method should make sure on their own not to call it with null...